babel-macro/src/test/macro.spec.ts

```diff
@@ -87,5 +94,20 @@ pluginTester({
         });
       `,
     },
+    "Supports TypeScript output": {
```

Comment:
we need to add all the javascript tests above in typescript too

Reply:
after f2f it is obvious this is not required, as the output should be tested as vanilla js